apps/rpc/src/modules/user/user-service.ts

```diff
@@ -261,6 +261,28 @@ export function getUserService(
       return user
     },
     async update(handle, userId, data) {
+      const result = UserWriteSchema.partial().safeParse(data)
+
+      if (!result.success) {
+        const error = result.error.errors[0]
```

Comment:
Can't you just do result.error.message to capture all errors?

```suggestion
result.error.message
````